packages/nx-plugin/src/utils/versions.ts

```diff
@@ -1,20 +1,26 @@
-import { readFileSync } from 'node:fs';
-import { join } from 'node:path';
+import { readFile } from 'node:fs/promises';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
 
-const workspaceRoot = join(__dirname, '../../');
-const projectsFolder = join(__dirname, '../../../');
-
-export const cpuNxPluginVersion = loadPackageJson(workspaceRoot).version;
-export const cpuModelVersion = loadPackageJson(
-  join(projectsFolder, 'cli'),
+const workspaceRoot = join(fileURLToPath(dirname(import.meta.url)), '../../');
+const projectsFolder = join(
+  fileURLToPath(dirname(import.meta.url)),
+  '../../../',
+);
+type PkgJson = { version: string };
+export const cpuNxPluginVersion = (await loadPackageJson(workspaceRoot))
+  .version;
+export const cpuModelVersion = (
+  await loadPackageJson(join(projectsFolder, 'cli'))
 ).version;
-export const cpuUtilsVersion = loadPackageJson(
-  join(projectsFolder, 'utils'),
+export const cpuUtilsVersion = (
+  await loadPackageJson(join(projectsFolder, 'utils'))
 ).version;
-export const cpuCliVersion = loadPackageJson(
-  join(projectsFolder, 'models'),
+export const cpuCliVersion = (
+  await loadPackageJson(join(projectsFolder, 'models'))
 ).version;
```

Comment:
Should use `cp` instead of `cpu` - [#branding](https://github.com/code-pushup/portal/wiki/Branding) :grin:

Reply:
renamed